modules/serverless-ossm-setup.adoc

```diff
@@ -1,12 +1,21 @@
-// Module included in the following assemblies:
-//
-// * serverless/networking/serverless-ossm.adoc
-
 [id="serverless-ossm-setup_{context}"]
-= Using {ProductShortName} with {ServerlessProductName}
+= Integrating {ProductShortName} with {ServerlessProductName}
+
+You can integrate {ProductShortName} with {ServerlessProductName} without using Kourier by completing the following procedure.
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} on your {product-title} cluster.
+* You have installed {ProductName}. {ServerlessProductName} with {ProductShortName} only is supported for use with {ProductName} version 2.0.5 or newer.
+
+[IMPORTANT]
+====
+Do not install the Knative Serving component before completing the following procedures. There are additional steps required when creating the `KnativeServing` custom resource defintion (CRD) to integrate Knative Serving with {ProductShortName}, which are not covered in the general Knative Serving installation procedure of the _Administration guide_.
+====
 
 .Procedure
 
+. Create a `ServiceMeshControlPlane` object in the `istio-system` namespace. If you want to use the mTLS functionality, this must be enabled for the `istio-system` namespace.
```

Comment:
Do you have an example `ServiceMeshControlPlane` object? Or, will folks know how to create one correctly?

Reply:
Do you have an example for configuring mTLS? Or, will folks know how to do this correctly?

Reply:
These are topics that should be covered in the core service mesh docs, since they're not Serverless specific and we only cover the integration in our docs.
e.g. https://docs.openshift.com/container-platform/4.7/service_mesh/v2x/ossm-create-smcp.html, https://docs.openshift.com/container-platform/4.7/service_mesh/v2x/ossm-security.html

We could look into providing links here once xrefs are available in modules, but I think for now it's OK without it.